manifests/server/grant_role.pp

```diff
@@ -39,7 +39,7 @@
     connect_settings => $connect_settings,
   }
 
-  if ! $connect_settings or empty($connect_settings) {
```

Comment:
Observed bit of confusing due to the DSL precedence as I thought negate (!) is for `$connect_settings or empty($connect_settings)` but I was wrong. Thanks to PDK console :p,
Thanks for the cleanup.